frontend/src/components/Register/Register.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import c from "./reg.module.css";
+import axiosInstance from "../../utils/axios";
+
+const Register: React.FC = () => {
+  const [firstName, setFirstName] = useState("");
```

Comment:
Umjesto da imamo 4 ova useStatea za firstname, lastname, email i password, možemo to prebacit u jedan objekt s obzirom da se odnose na istu stvar. Dakle da imamo userData state unutar kojeg su svi ovi ostali atributi. Javi se ako ne budeš znala

Reply:
Mozemo to, a i mozemo neki library za forme koristit tipa rhf
Sad neznan jel ima potrebe za tin jer nevirujen da cemo imat prekompleksne forme pa mislin da je objekt najbolje rjesenje, ali eto cisto prijedlog